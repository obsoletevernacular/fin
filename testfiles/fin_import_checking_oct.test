Transaction ID,Posting Date,Effective Date,Transaction Type,Amount,Check Number,Reference Number,Description,Transaction Category,Type,Balance
"20171031 222222 2,222 222,222,222,222",10/31/17,11/1/17,Debit,-15.69,,20,POS Transaction PLACE         LOCATION,,Debit Card,15.69
"20171031 222222 2,222 222,222,222,222",10/31/17,10/31/17,Credit,10,,21,Credit Dividend,,,25.69
"20171031 222222 2,222 222,222,222,222",10/31/17,10/31/17,Debit,-1.69,,22,ACH Debit DESCRIPTION - BILL PAYMT,,ACH,24
"20171030 222222 2,222 222,222,222,222",10/30/17,10/31/17,Debit,-1.15,,23,POS Transaction PLACE      LOCATION,,Debit Card,22.85
"20171030 222222 2,222 222,222,222,222",10/30/17,10/31/17,Debit,-1.15,,24,POS Transaction PLACE      LOCATION,,Debit Card,21.7
"20171030 222222 2,222 222,222,222,222",10/30/17,10/30/17,Debit,-2.14,,25,POS Transaction PLACE      LOCATION,,Debit Card,19.56
"20171030 222222 2,222 222,222,222,222",10/30/17,10/30/17,Debit,-1.15,,26,POS Transaction PLACE      LOCATION,,Debit Card,18.41
"20171030 222222 2,222 222,222,222,222",10/30/17,10/30/17,Debit,-1.15,,27,POS Transaction PLACE      LOCATION,,Debit Card,17.26
"20171027 222222 2,222 222,222,222,222",10/28/17,10/28/17,Debit,-1.15,,28,POS Transaction PLACE      LOCATION,AUTOCAT,Debit Card,16.11