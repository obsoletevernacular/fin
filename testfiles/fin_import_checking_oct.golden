1 files processed and stored to .default.obj
fin_import_checking_oct
    transactions: 9
    transactions_in: 1
    transactions_out: 8
    sum: -15.27
    sum_in: 10.00
    sum_out: -25.27
    avg: -1.70
1 files processed. summary report below:
summary
    transactions: 9
    transactions_in: 1
    transactions_out: 8
    sum: -15.27
    sum_in: 10.00
    sum_out: -25.27
    avg: -1.70
